### PR TITLE
replace moment js with date-fns

### PR DESCRIPTION
**Pull request by davemcorwin**
There is no direct equivalent for moment's `.calendar()`. For now, on the bill page, I am showing the date and time with no fancy 'Today', 'Yesterday', etc...

### CODE REVIEW

File: app/javascript/src/scenes/bill/components/Content.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
 // @flow
 import React, { Component } from 'react'
 import { createFragmentContainer, graphql } from 'react-relay'
-import moment from 'moment'
+import { default as parseDate } from 'date-fns/parse'
```
Comment (tycobbb) [inline app/javascript/src/scenes/bill/components/Content.js]: Can these default imports be aliased using the default import syntax?

```
import parseDate from 'date-fns/parse'
```

File: app/javascript/src/scenes/bill/components/Content.js [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+import differenceInHours from 'date-fns/difference_in_hours'
+import { default as dateFormat } from 'date-fns/format'
```
Comment (tycobbb) [inline app/javascript/src/scenes/bill/components/Content.js]: Unless you're opposed, can we call this `formatDate`? I try to name all methods starting with a verb.

Comment (davemcorwin) [reply app/javascript/src/scenes/bill/components/Content.js]: 👍

File: app/javascript/src/scenes/bill/components/Content.js (hunk 1 above)
Comment (davemcorwin) [reply app/javascript/src/scenes/bill/components/Content.js]: yes, but with generic names like `parse` and `format` I wanted to rename them. I'll put them in our own module and that will help with this.

**Comment by davemcorwin** [comment]
@wzrad this is ready for review again, sorry for the delay.

**Comment by tycobbb** [comment]
awesome, looks great. build error looks totally unrelated and probably my fault, so merging.

**Pull request merged**